pkg/client.go

```diff
@@ -244,11 +243,10 @@ func (c *Client) MakeAPICall(verb string, r *Response, data []byte) error {
 	}
 	resp.Body.Close()
 	respString := string(respBytes)
```

Comment:
And we don't need `respString` either now! If we pass `respBytes` to `fmt.Errorf` at line 250 using the `%q` verb, it will be printed as a quoted string.